test/tests/meshgenerators/poly2tri_mesh_generator/poly2tri_nested.i

```diff
@@ -0,0 +1,71 @@
+[Mesh]
+  [inner_square_sbd_0]
+    type = GeneratedMeshGenerator
+    dim = 2
+    nx = 3
+    ny = 3
+    xmin = -0.4
+    xmax = 0.4
+    ymin = -0.4
+    ymax = 0.4
+  []
+  [inner_square]
+    type = SubdomainIDGenerator
```

Comment:
Minor: The above `GeneratedMeshGenerator` has a parameter `subdomain_ids` that can be used to assign subdomain ids of each element.

Reply:
Yeah.  This seemed more flexible for debugging, though; I can run different resolutions with e.g. `Mesh/inner_square_sbd_0/nx=5` on the command line without having to resize a `Mesh/inner_square_sbd_0/subdomain_ids` vector to fit.